fe/fe-core/src/main/java/com/starrocks/planner/stream/StreamAggNode.java

```diff
@@ -0,0 +1,94 @@
+// This file is licensed under the Elastic License 2.0. Copyright 2021-present, StarRocks Inc.
+
+package com.starrocks.planner.stream;
+
+import com.starrocks.analysis.AggregateInfo;
+import com.starrocks.analysis.Expr;
+import com.starrocks.planner.PlanNode;
+import com.starrocks.planner.PlanNodeId;
+import com.starrocks.sql.optimizer.operator.physical.stream.IMTInfo;
+import com.starrocks.thrift.TExplainLevel;
+import com.starrocks.thrift.TExpr;
+import com.starrocks.thrift.TPlanNode;
+import com.starrocks.thrift.TPlanNodeType;
+import com.starrocks.thrift.TStreamAggregationNode;
+import org.apache.commons.collections.CollectionUtils;
+
+import java.util.List;
+import java.util.stream.Collectors;
+
+public class StreamAggNode extends PlanNode {
```

Comment:
Could you add the IMT logo? It's easy to confuse `stream` with some current features

Reply:
All node of MV operator would be prefixed with `Stream`, and placed at `stream` directory.

Reply:
The new aggregation implementation should be `SortAggregate`, instead of `StreamAggregate`

Reply:
Which word `Stream` or `Streaming` is better? we should unify the verbs in be and fe.